### PR TITLE
Deal with sudden postgres upgrade without a path for upgrading

### PR DESCRIPTION
**Pull request by yakloinsteak**
We shouldn't just upgrade the database major version without helping people upgrade.

This PR also unifies the Silcon/ARM devs with everybody else so we can have one image. We also use the standard postgres image with is safer.

Testing involves running the script in this PR and commenting out any overrides of the image for the database you use locally.

### CODE REVIEW

**Review by eanders — APPROVED** [review]
Looks great.  Thanks!  I'm holding off merging until I've had a chance to test locally.

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -115,8 +115,29 @@ services:
...
+      - 5432
+    expose:
+      - 5432
+
+  # This should no longer need to be overridden for ARM
+  # processors
   db:
-    image: postgis/postgis:13-3.3-alpine
+    build:
+      context: docker/pg13
```
Comment (eanders) [inline docker/docker-compose.yml:142]: What re the chances we can make this postgis.  That's the main reason I have an override (I think) `image: gangstead/postgis:13-3.1-arm`

**Comment by eanders** [comment]
This didn't work for me, but maybe I'm missing something.  Let's connect at some point.

File: docker/docker-compose.yml (hunk 1 above)
Comment (yakloinsteak) [reply docker/docker-compose.yml:142]: The Dockerfile installs postgis, and I confirmed that postgis works for me locally.

**Pull request merged**